src/Cohesity.Powershell/Scripts/Vlan/Get-CohesityVlan.ps1

```diff
@@ -0,0 +1,80 @@
+#### USAGE ####
+#	********************** Using Function *********************
+#   Get-CohesityVlan
+#   Get-CohesityVlan -allUnderHierarchy false -skipPrimaryAndBondIface false -tenantIds testOrg/
+#   Get-CohesityVlan -allUnderHierarchy false -skipPrimaryAndBondIface true
+#   Get-CohesityVlan -VlanId 0
+###############
+
+function Get-CohesityVlan {
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false)]
+        [Int64]$VlanId,
+        # Specifies if objects of all the tenants under the hierarchy of the logged in user’s organization should be returned.
+        [Parameter(Mandatory = $false)]
+        $allUnderHierarchy = $null,
+        # Filter interfaces entries which are primary interface or bond interfaces.
+        [Parameter(Mandatory = $false)]
+        $skipPrimaryAndBondIface = $null,
+        # TenantIds contains ids of the tenants for which objects are to be returned.
+        [Parameter(Mandatory = $false)]
+        [string[]]$tenantIds = $null
+
+    )
+    Begin {
+        if (-not (Test-Path -Path "$HOME/.cohesity")) {
```

Comment:
If the file exists but token has expired, how'd we handle that error ?

Reply:
That will throw an exception 401(unauthorized) error

Reply:
Got it, will require a generic fix in invoke-restAPI lets talk to bijay about this.